app-antivirus/fangfrisch/fangfrisch-1.0.1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_COMPAT=( python3_{7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Update and verify unofficial Clam Anti-Virus signatures"
+HOMEPAGE="https://github.com/rseichter/fangfrisch https://pypi.org/project/fangfrisch/"
+SRC_URI="https://github.com/rseichter/fangfrisch/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND=">=dev-python/requests-2.22.0[${PYTHON_USEDEP}]
+	>=dev-python/sqlalchemy-1.3.11[${PYTHON_USEDEP}]"
+RDEPEND="${DEPEND}"
+
+distutils_enable_tests unittest
+
+function python_prepare_all() {
+	sed -i -E -e 's/(SQLAlchemy >= 1.3.1)3/\11/' setup.py || die
+	distutils-r1_python_prepare_all
+}
+
+function python_install_all() {
+	insinto etc
```

Comment:
`/etc` and so on below

Reply:
I had actually thought using relative paths would constitute "being nice":
```
→ grep -lr 'insinto etc' /var/db/repos/gentoo/
/var/db/repos/gentoo/net-vpn/vtun/vtun-3.0.3-r1.ebuild
/var/db/repos/gentoo/net-vpn/vtun/vtun-3.0.3.ebuild
/var/db/repos/gentoo/media-sound/flack/flack-2.0.6.ebuild
/var/db/repos/gentoo/app-admin/gentoo-rsync-mirror/gentoo-rsync-mirror-1.0-r5.ebuild
/var/db/repos/gentoo/app-admin/github-backup-utils/github-backup-utils-2.18.0.ebuild
/var/db/repos/gentoo/app-admin/github-backup-utils/github-backup-utils-2.13.2.ebuild
/var/db/repos/gentoo/app-admin/github-backup-utils/github-backup-utils-2.13.0.ebuild
/var/db/repos/gentoo/app-admin/github-backup-utils/github-backup-utils-2.11.1-r1.ebuild
/var/db/repos/gentoo/app-admin/github-backup-utils/github-backup-utils-2.10.0-r1.ebuild
/var/db/repos/gentoo/app-admin/github-backup-utils/github-backup-utils-2.11.1.ebuild
/var/db/repos/gentoo/app-admin/github-backup-utils/github-backup-utils-2.17.0.ebuild
/var/db/repos/gentoo/app-admin/github-backup-utils/github-backup-utils-2.11.0.ebuild
/var/db/repos/gentoo/app-admin/github-backup-utils/github-backup-utils-2.14.2.ebuild
/var/db/repos/gentoo/app-admin/github-backup-utils/github-backup-utils-2.16.1.ebuild
/var/db/repos/gentoo/app-admin/github-backup-utils/github-backup-utils-2.11.0-r1.ebuild
/var/db/repos/gentoo/app-admin/github-backup-utils/github-backup-utils-2.10.0.ebuild
/var/db/repos/gentoo/app-admin/github-backup-utils/github-backup-utils-2.19.1.ebuild
/var/db/repos/gentoo/app-admin/github-backup-utils/github-backup-utils-2.11.2-r1.ebuild
/var/db/repos/gentoo/app-admin/github-backup-utils/github-backup-utils-2.15.1.ebuild
/var/db/repos/gentoo/app-admin/github-backup-utils/github-backup-utils-2.11.2.ebuild
/var/db/repos/gentoo/app-admin/github-backup-utils/github-backup-utils-2.15.0.ebuild
/var/db/repos/gentoo/net-fs/autofs/autofs-5.1.5-r2.ebuild
/var/db/repos/gentoo/net-fs/autofs/autofs-5.1.6.ebuild

```